cfgov/regulations3k/regdown.py

```diff
@@ -102,6 +144,17 @@ def extendMarkdown(self, md, md_globals):
         del md.parser.blockprocessors['olist']
 
 
+class PseudoFormPattern(Pattern):
+    """ Return a <span class="pseudo-form"></span> element for matches of the
+    given pseudo-form pattern. """
+
+    def handleMatch(self, m):
+        el = util.etree.Element('span')
+        el.set('class', 'pseudo-form')
```

Comment:
How would you feel about using the class `regdown-form` instead of `pseudo-form`?